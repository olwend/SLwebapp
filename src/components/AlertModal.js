import React from 'react'

import { Modal, Button } from 'react-bootstrap'

const AlertModal = props =>
    <Modal {...props} size="lg" aria-labelledby="contained-modal-title-vcenter" centered>
      <Modal.Header closeButton>
        <Modal.Title id="contained-modal-title-vcenter">
          { props.err ? props.err.header : '' }
        </Modal.Title>
      </Modal.Header>
      <Modal.Body>
        <p>
          { props.err ? props.err.message : '' }
        </p>
      </Modal.Body>
      <Modal.Footer>
        <Button onClick={ props.onHide }>Close</Button>
      </Modal.Footer>
    </Modal>

export default AlertModal
