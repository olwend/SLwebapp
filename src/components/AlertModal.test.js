import React from 'react'

import { shallow } from 'enzyme'

import AlertModal from './AlertModal'

test('Renders properly', () => expect(shallow(<AlertModal />)).toMatchSnapshot())
