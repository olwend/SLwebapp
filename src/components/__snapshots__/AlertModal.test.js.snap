// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders properly 1`] = `
<Modal
  animation={true}
  aria-labelledby="contained-modal-title-vcenter"
  autoFocus={true}
  backdrop={true}
  centered={true}
  dialogAs={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "displayName": "ModalDialog",
      "render": [Function],
    }
  }
  enforceFocus={true}
  keyboard={true}
  restoreFocus={true}
  show={false}
  size="lg"
>
  <ModalHeader
    closeButton={true}
    closeLabel="Close"
  >
    <ModalTitle
      id="contained-modal-title-vcenter"
    />
  </ModalHeader>
  <ModalBody>
    <p />
  </ModalBody>
  <ModalFooter>
    <Button
      active={false}
      disabled={false}
      variant="primary"
    >
      Close
    </Button>
  </ModalFooter>
</Modal>
`;
